x-pack/legacy/plugins/canvas/i18n/functions/dict/formatnumber.ts

```diff
@@ -9,24 +9,22 @@ import { formatnumber } from '../../../canvas_plugin_src/functions/common/format
 import { FunctionHelp } from '../function_help';
 import { FunctionFactory } from '../../../types';
 import { NUMERALJS } from '../../constants';
+// import { getDocumentationLinks } from '../../../public/lib/documentation_links';
```

Comment:
It looks like we can't access doc links from our i18n strings since they don't live in `/public`, but I think we should still figure out a way to insert a link into the help text here. This isn't a blocker for this PR. I'll file an issue to solve this later.